client/app/account/billing/mean/add/billing-mean-add.controller.js

```diff
@@ -223,7 +223,7 @@ angular.module("Billing.controllers").controller("Billing.controllers.Mean.Add",
             })
             .then(() => User.getUser())
             .then((user) => {
-                $scope.isFrenchCustomer = user.ovhSubsidiary === "FR";
+                $scope.customerIsFromFrance = user.billingCountry === "FR";
```

Comment:
Why not `customerIsChargedInFrance` or something more related to billing? As it is the renaming doesn't transmit more information.

Reply:
Actually not sure that `billingCountry` is the billing country, it just says `country` in the API, so I chose `fromFrance` because it's vague enough 😄

Reply:
Makes sense :man_cartwheeling: